projects/npm-tools/packages/npm-scripts/src/utils/expandGlobs.js

```diff
@@ -9,6 +9,7 @@ const path = require('path');
 const getRegExpForGlob = require('./getRegExpForGlob');
 
 const DEFAULT_OPTIONS = {
+	baseDir: '.',
```

Comment:
Would be nice to see these changes covered in [the tests](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/packages/npm-scripts/test/utils/expandGlobs.js).